### ISSUE TITLE
Parallel Project?

### ISSUE THREAD

**Issue by mschrader15** [dvargas92495/generate-roam-site-action#15]
Have y'all seen this Roam -> Website project: https://github.com/mathieudutour/gatsby-digital-garden ?

**Comment by caffo** [comment]
I found it after we started working on this :)

**Comment by caffo** [comment]
Adding to it, I think what we have here is simpler and I like that.

**Comment by mschrader15** [comment]
I agree. The ability to have ~page-~ block-level control over what is displayed/not displayed inside of Roam Research really sets this project apart.

There are portions of both projects that could probably be extracted into common libraries like @dvargas92495 
 said in https://github.com/dvargas92495/generate-roam-site-action/issues/13#issuecomment-753519779, specifically the Roam export and Markdown conversion

**Comment by dvargasfuertes** [comment]
I actually used it as a reference to build this one ;). It hadn't been maintained in awhile so I just went forward starting from scratch and referencing it instead of building on top of it

**Comment by Stvad** [comment]
it already extracts the things like fetching/parsing references/etc in separate packages btw. so I think that can be reused straight away

**Comment by Stvad** [comment]
just discovered a whole bunch of similar/fascinating things at https://github.com/MaggieAppleton/digital-gardeners
